src/langcheck/metrics/eval_clients/_litellm.py

```diff
@@ -27,6 +28,10 @@ def __init__(
         embedding_model: str | None = None,
         *,
         use_async: bool = False,
+        use_reasoning_summary: bool = False,
+        reasoning_effort: ReasoningEffort = "medium",
+        reasoning_summary: Literal["auto", "concise", "detailed"]
+        | None = "auto",
```

Comment:
Related to [this comment](https://github.com/citadel-ai/langcheck/pull/208#discussion_r2266721125):

We can even remove `use_reasoning_summary` from the arguments since we can use `reasoning_summary` as a flag? In that case we need to make `reasoning_summary = None` by default

Reply:
Actually completions API also has `reasoning_effort` option, so let's keep `use_reasoning_summary` argument for clarity.